.trunk/trunk

```diff
@@ -0,0 +1,186 @@
+#!/bin/bash
```

Comment:
This script doesn't conform to our bash style as a whole, see other bash scripts for equivalents.

All bash scripts live in `make/buf/scripts` and are controlled by the `make` system.

All installation is controlled by the `make` system.